cli/src/main/java/com/devonfw/tools/ide/tool/ToolCommandlet.java

```diff
@@ -82,18 +82,20 @@ public final Set<Tag> getTags() {
   @Override
   public void run() {
 
-    runTool(null, this.arguments.asArray());
+    runTool(false, null, this.arguments.asArray());
   }
 
   /**
    * Ensures the tool is installed and then runs this tool with the given arguments.
    *
+   * @param isBackgroundProcess {@code true}, the process of the command will be run as background process,
+   *        {@code false} otherwise it will be run as foreground process.
    * @param toolVersion the explicit version (pattern) to run. Typically {@code null} to ensure the configured version
    *        is installed and use that one. Otherwise, the specified version will be installed in the software repository
    *        without touching and IDE installation and used to run.
    * @param args the commandline arguments to run the tool.
    */
-  public void runTool(VersionIdentifier toolVersion, String... args) {
+  public void runTool(boolean isBackgroundProcess, VersionIdentifier toolVersion, String... args) {
```

Comment:
to reduce merge conflicts of other developers, it is good if you keep the original method signature and delegate to the new one with `false` keeping the existing behavior.

Reply:
Done